packages/web-components/@sfx/search-box/test/unit/search-box.test.ts

```diff
@@ -95,7 +95,7 @@ describe('SearchBox Component', () => {
       const detail = 'inputText';
       const inputEvent = new CustomEvent('some-test-type', { detail });
       const updateSearchTermValue = stub(searchbox, 'updateSearchTermValue');
-  
+
```

Comment:
It may have been your formatter doing this, but since this is an unrelated file, please revert this change.